test/unit_tests/unittest_decode_exception.cpp

```diff
@@ -1,27 +0,0 @@
-// Copyright (c) 2020 Pilz GmbH & Co. KG
-//
-// This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU Lesser General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU Lesser General Public License for more details.
-//
-// You should have received a copy of the GNU Lesser General Public License
-// along with this program.  If not, see <https://www.gnu.org/licenses/>.
-
-#include <gtest/gtest.h>
-#include <psen_scan/decode_exception.h>
-
-using namespace psen_scan;
-
-namespace psen_scan_test
-{
-TEST(DecodeExceptionTest, new_build_ros_message_exception)
```

Comment:
Why is this gone?

Reply:
because:
https://github.com/PilzDE/psen_scan/blob/e55a5a644befa3b63477084b79afc3d266c9f7b6/test/unit_tests/unittest_msg_decoder.cpp#L120